package.json

```diff
@@ -62,6 +62,7 @@
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
+    "less": "^2.7.1",
```

Comment:
FYI... less is needed by the sass compiler in wrbpack iirc.

On Thu, May 12, 2016, 09:31 Alexander Ose <notifications@github.com> wrote:

> In package.json
> <https://github.com/department-of-veterans-affairs/healthcare-application/pull/37#discussion_r63052256>
> :
>
> > @@ -62,6 +62,7 @@
> >      "karma-phantomjs-launcher": "^1.0.0",
> >      "karma-sourcemap-loader": "^0.3.7",
> >      "karma-webpack": "^1.7.0",
> > +    "less": "^2.7.1",
>
> This was an unrelated issue with npm shrinkwrap. It wouldn't work because
> of an unresolved peer dependency with less (and also this: npm/npm#6565
> <https://github.com/npm/npm/issues/6565>).
>
> In npm install, it's just a warning, but in shrinkwrap it's an error.
> Hence the need to explicitly resolve it. If there's a better way around
> this issue, I'd love to know...
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/department-of-veterans-affairs/healthcare-application/pull/37/files/1e52cbc1e9ae36cd530fa65633786ebf5ed74380#r63052256>
>